gqueries/output_elements/output_series/user_uploaded_profiles/industry_chemicals_heat_curve.gql

```diff
@@ -49,7 +49,7 @@
           V(
             FILTER(
             ALL(),
-            "hydrogen && (hydrogen.profile == :industry_chemicals_heat)"
+            "hydrogen && (hydrogen.group == :industry_chemicals_heat)"
```

Comment:
This is similar to the issue with heat in agriculture and should be addressed in a similar way.